### ISSUE TITLE
Very low FPS

### ISSUE THREAD

**Issue by xedro98** [wpine215/msfs-a320neo#229]
As soon as I install this and start flying the A320 neo, my fps drops almost 30-40% but it is all fine once I uninstall this mode, any help would be appreciated. 

Thanks!

**Comment by Brenchen** [comment]
This may not help directly, but for more info:
- Vanilla addon?  As in you're not using different liveries?  If you are, have you tried to switch back to the default livery?
- Does the FPS suffer when you uninstall and run the default MSFS 320?
- What airport are you doing this in?  Can you try another lower density and default airport?
- What graphics setting are you using?
- Can you take a video explaining (ideally talk us through) what the steps you're doing to get to the point of a dropped frames?

This sounds like an isolated event, since we don't have your computer in front of us, we can only try to give you some ideas and ways to troubleshoot.  Once you find out more what's going on, or are able to give the devs some steps to reproduce your problem on their machines, they (or you) can try to fix it.

Basically, try to eliminate ANY distractions - use as many defaults as possible (airfield, plane, livery, scenery, weather, timing etc.).

**Comment by Curtis-VL** [comment]
I'm going to close this issue as it's not a direct bug report with a specific system on the aircraft, please bring it up in our Discord though and review the comment from @Brenchen for some help! :)
